src/js/components/TaskTable.js

```diff
@@ -39,9 +40,55 @@ class TaskTable extends React.Component {
   }
 
   getStatusValue(task) {
+    let taskHealth = this.getTaskHealth(task);
+    if (taskHealth === true) {
```

Comment:
Yeah, it's a confusing API - we only show task health as a status if `taskHealth` is defined. If it's undefined, we use the `state` property to infer a status (like `Killing`). This is [covered in the tests](https://github.com/dcos/dcos-ui/pull/955/files#diff-f1869143cacab7fd5bff4b5dbe2094f9R145) but maybe we should add a comment here to ease readability.